examples/with-serverless/README.md

```diff
@@ -0,0 +1,22 @@
+# nextjs-serverless
```

Comment:
I think the example should be called `with-serverless-framework`, there are many ways do to serverless (eg deploying to Now) and this community plugin is just one of them.

Reply:
makes sense. Fixed ✅